### PR TITLE
[Bugfix/schema] Changes for validate.js

### PR DESCRIPTION
**Pull request by KseniiaSokolovaEpam**
There are changes in skin-schema.json , skin.json, tv-skin.json for prevent errors after runing 
```
find . -type f -name "*skin.json" | xargs -I REPL node validate.js skin-schema.json REPL
```

### CODE REVIEW

File: skin-schema.json [hunk 1]
```diff
@@ -716,6 +716,36 @@
...
+          "default": "If no value is provided for vrNotification, vrNotification will default to true."
+        },
+        "vrIcon": {
+          "description": "Vr icon should be  shown",
+          "type": "boolean",
+          "default": "If no value is provided for vrIcon, vrIcon will default to false."
+        }
+      }
+    },
+    "animationDurations": {
```
Comment (jpfranco) [inline skin-schema.json]: This should be `animationDuration`

File: skin-schema.json [hunk 2]
```diff
@@ -716,6 +716,36 @@
...
+      "description": "Sets duration for vr animation",
+      "type": "object",
+      "vrNotification": {
+        "description": "Duration for vrNotification",
+        "type": "boolean",
+        "default": 5
```
Comment (jpfranco) [inline skin-schema.json]: The type is `boolean` but the default is a number

File: skin-schema.json [hunk 3]
```diff
@@ -716,6 +716,36 @@
...
+      },
+      "vrIcon": {
+        "description": "Duration for vrIcon",
+        "type": "boolean",
+        "default": 4
```
Comment (jpfranco) [inline skin-schema.json]: The type is `boolean` but the default is a number

File: skin-schema.json [hunk 4]
```diff
@@ -1174,6 +1204,26 @@
           "type": "string",
           "default": "icon icon-auto"
         }
+      },
+      "multiAudio": {
+        "description": "Set the styling for the multiaudio",
```
Comment (jpfranco) [inline skin-schema.json]: Can we change this for "Set the styling for the multiaudio icon"?

File: skin-schema.json [hunk 5]
```diff
@@ -1226,7 +1276,7 @@
     "buttonNames": {
       "description": "These are the only buttons currently supported by the UI SDK.",
       "additionalProperties": false,
-      "enum": ["playPause", "volume", "timeDuration", "flexibleSpace", "discovery", "fullscreen", "stereoscopic",  "moreOptions", "arrowsBlack", "logo", "share", "closedCaption", "bitrateSelector", "quality", "live", "multiAudio"]
```
Comment (jpfranco) [inline skin-schema.json]: Is `bitrateSelector` no longer needed?

File: skin-schema.json [hunk 6]
```diff
@@ -716,6 +716,36 @@
         }
       }
     },
+    "isVrAnimationEnabled": {
```
Comment (jpfranco) [inline skin-schema.json]: I think these properties (`isVrAnimationEnabled`, `animationDuration`) should be inside a `vr` (or similarly named) section.

File: skin-schema.json [hunk 7]
```diff
@@ -716,6 +716,36 @@
...
+      "description": "Shows if vr inform animation is enabled",
```
Comment (jpfranco) [inline skin-schema.json]: Can we describe the animation that this setting controls? From reading this it's not clear to the user what will happen if this is enabled

File: skin-schema.json (hunk 6 above)
Comment (jpfranco) [reply skin-schema.json]: Also, I think `isVrAnimationEnabled` sound like it should be a boolean value. Should this be something like `vrAnimation` and then have a separate `enabled` property?

File: skin-schema.json [hunk 8]
```diff
@@ -716,6 +716,36 @@
...
+      "type": "object",
+      "properties": {
+        "vrNotification": {
+          "description": "Vr notification should be shown",
+          "type": "boolean",
+          "default": "If no value is provided for vrNotification, vrNotification will default to true."
+        },
+        "vrIcon": {
+          "description": "Vr icon should be  shown",
```
Comment (aeng7) [inline skin-schema.json]: Extra space between be and shown

File: skin-schema.json [hunk 9]
```diff
@@ -716,6 +716,36 @@
...
+          "type": "boolean",
+          "default": "If no value is provided for vrIcon, vrIcon will default to false."
+        }
+      }
+    },
+    "animationDurations": {
+      "description": "Sets duration for vr animation",
```
Comment (aeng7) [inline skin-schema.json]: Should this be vrAnimation? In other places, like 3 lines below, I see "vrNotification." Let's shoot for consistency in our docs

File: skin-schema.json [hunk 10]
```diff
@@ -716,6 +716,36 @@
...
+      "type": "object",
+      "vrNotification": {
+        "description": "Duration for vrNotification",
```
Comment (aeng7) [inline skin-schema.json]: Let's separate VR out in descriptions for consistency. This should be something like:

Duration for VR Notification

File: skin-schema.json (hunk 1 above)
Comment (KseniiaSokolovaEpam) [reply skin-schema.json]: variable name is animationDurationS :-) because it is object with several durations.

File: skin-schema.json (hunk 2 above)
Comment (KseniiaSokolovaEpam) [reply skin-schema.json]: \+

File: skin-schema.json (hunk 3 above)
Comment (KseniiaSokolovaEpam) [reply skin-schema.json]: \+

File: skin-schema.json (hunk 4 above)
Comment (KseniiaSokolovaEpam) [reply skin-schema.json]: \+

File: skin-schema.json (hunk 5 above)
Comment (KseniiaSokolovaEpam) [reply skin-schema.json]: I tried to find bitrateSelector in 
- html5-skin
- tml5-video-plugins
- html5-ad-plugins
- html5-analytics-plugins
- video-plugins-internal
- html5-common
- mjolnir

And bitrateSelector doesn not exist there. 
I found bitrateSelector in https://github.com/ooyala/native-skin but it is for sdk.

File: skin-schema.json (hunk 8 above)
Comment (KseniiaSokolovaEpam) [reply skin-schema.json]: \+

File: skin-schema.json (hunk 10 above)
Comment (KseniiaSokolovaEpam) [reply skin-schema.json]: \+

File: skin-schema.json (hunk 9 above)
Comment (KseniiaSokolovaEpam) [reply skin-schema.json]: \+

File: skin-schema.json (hunk 7 above)
Comment (KseniiaSokolovaEpam) [reply skin-schema.json]: \+

File: skin-schema.json (hunk 1 above)
Comment (jpfranco) [reply skin-schema.json]: Oh ok, I see. Looks good then.

File: skin-schema.json (hunk 5 above)
Comment (jpfranco) [reply skin-schema.json]: @KseniiaSokolovaEpam the skin config is shared between web and sdks, so we shouldn't remove anything that's used by the sdks.

File: skin-schema.json (hunk 6 above)
Comment (jpfranco) [reply skin-schema.json]: Oh ok, if this was already there then I think we can keep it.

**Comment by aeng7** [comment]
LGTM outside of @jpfranco's comments.

File: skin-schema.json (hunk 5 above)
Comment (KseniiaSokolovaEpam) [reply skin-schema.json]: Oh, ok, I got it. Thank you! I will return it back.

Comment (KseniiaSokolovaEpam) [reply skin-schema.json]: \+

**Pull request merged**